test/fuzz/lua/pickle_unpack_test.lua

```diff
@@ -0,0 +1,15 @@
+local pickle = require("pickle")
+local luzer = require("luzer")
+
+local function TestOneInput(buf)
+    local ok, unpacked = pcall(pickle.unpack, buf)
```

Comment:
This always returns `false` since `pickle.unpack()` expects a string as the second argument.